src/components/Navigation.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Navigation.scss';
+
+function Navigation() {
+    return (
+        <nav className="Navigation">
```

Comment:
The nav does not need a navigation class because it's the only nav component on the page.